package.json

```diff
@@ -44,30 +44,30 @@
 		"prepare": "npm run build && npm rebuild --build-from-source gl"
 	},
 	"dependencies": {
-		"@assemblyscript/loader": "^0.19.3",
+		"@as-pect/cli": "^7.0.7",
+		"@assemblyscript/loader": "^0.21.3",
 		"@lume/as-loader": "^0.18.10",
 		"aswebglue": "lume/aswebglue#653897e69c6d7f74a7dcf71cdb5e4cea1d3f322d"
 	},
 	"devDependencies": {
-		"@as-pect/cli": "^6.2.4",
-		"@types/node": "^15.12.4",
-		"assemblyscript": "^0.19.3",
-		"deasync": "^0.1.21",
-		"glfw-raub": "^4.6.0",
-		"live-server": "^1.2.1",
+		"@types/node": "^18.7.14",
+		"assemblyscript": "^0.21.3",
+		"deasync": "^0.1.28",
```

Comment:
Nice, looks like we can remove some dependencies that were needed for `importSync`.